RELEASE_NOTES.md

```diff
@@ -1,165 +1,165 @@
-### 4.0.0-beta-2
```

Comment:
I'm not sure if readme fake parser cares about ### vs ##, something to confirm.

Reply:
I tried it, and it seems to be generating the `Meta.props` file correctly.

If it doesn't let me know, and we will just point to the Release notes, inside the source repository.